packages/lib-classifier/src/store/subjects/Subject/FreehandLineReductions/FreehandLineReductions.js

```diff
@@ -11,29 +11,22 @@ const FreehandLineReductions = types
     workflowId: types.string
   })
   .volatile(self => ({
-    isUsed: types.boolean,
+    isUsed: types.array(types.boolean),
```

Comment:
Similar to my previous review - is this actually used anywhere? What is it for?

Reply:
Fixed